integration-test/src/it/relocation-plugin/verify.groovy

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright (C) 2012 Red Hat, Inc. (jcasey@redhat.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+def pomFile = new File( basedir, 'pom.xml' )
+System.out.println( "Slurping POM: ${pomFile.getAbsolutePath()}" )
+
+def pom = new XmlSlurper().parse( pomFile )
+
+def dependency = pom.dependencies.dependency.find { it.groupId.text() == "junit" }
+assert dependency != null
+assert dependency.artifactId.text() == "junit"
+assert dependency.version.text() == "4.1"
+
+dependency = pom.dependencies.dependency.find { it.groupId.text() == "org.slf4j" }
+assert dependency != null
+assert dependency.artifactId.text() == "slf4j-api"
+assert dependency.version.text() == "1.7.30"
+
```

Comment:
The checks above do not accurately check that the relocations for junit/slf4j worked - they verify the dependencies but _not_ the artifact items in maven-dependency-plugin

Reply:
These first 8 lines or so could be deleted. It was just checking the relocation of the dependency first before checking the the same relocation was applied to the plugin/configuration sections.